examples/volume_api500.rb

```diff
@@ -37,6 +37,7 @@
   data(store_serv_properties)
   storage_pool 'CPG-SSD-AO'
   storage_system 'ThreePAR-1'
+  action :create_if_missing
```

Comment:
you don't need to specify here explicitly..if you make this action default here:https://github.com/HewlettPackard/oneview-chef/blob/master/resources/volume.rb#L23

Reply:
done